src/anchor-click-interceptor.js

```diff
@@ -59,6 +60,14 @@ function maybeExpandUrlParams(ampdoc, e) {
     'CLICK_Y': () => {
       return e.pageY;
     },
+    '3PANALYTICS': (frameType, key) => {
+      const responses = ResponseMap.get(ampdoc, frameType,
```

Comment:
Why do you need responseMap?  Idea was to use closure to keep track of data where message listener would update ampDoc.getAnchorClickListenerBinding() value.